### PR TITLE
Clarify that the installer is an internal tool in the installer readme

### PR DESCRIPTION
**Pull request by lucasvaltl**
## Description
This clarifies that the installer is an internal tool in the installer readme. It clarifies this in the installer `readme.md` file and moves the usage instructions into the `/docs` folder. 

## How to test
- look at changes :) 

## Release Notes

```release-note
NONE
```

### CODE REVIEW

File: install/installer/docs/overview.md [hunk 1]
```diff
@@ -0,0 +1,589 @@
...
+  <p align="center">Always ready-to-code.</p>
+</p>
+
+# Installer
+
+The best way to get started with Gitpod is by using our recommended & default installation method [described in our documentation](https://www.gitpod.io/docs/self-hosted/latest/getting-started). In fact, our default installation method actually wraps this installer into a UI that helps you manage, update and configure Gitpod in a streamlined way. This document describes how to use the installer directly.
+
+> The installer is an internal tool and as such not expected to be used by those external to Gitpod.
+
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-908a85?logo=gitpod)](https://gitpod.io/from-referrer/)
```
Comment (Pothulapati) [inline install/installer/docs/overview.md:15]: nit: But I wonder if we need this badge anymore in this docs file? It makes sense in the README to open a gitpod instance with that code but not here? 🤔

**Review by Pothulapati — APPROVED** [review]
LGTM, apart from the nit!

/hold on that

File: install/installer/docs/overview.md [hunk 2]
```diff
@@ -0,0 +1,589 @@
...
+
+**IMPORTANT**: this does not create the namespace, so you will need to create
+that separately. This is so that uninstallation of Gitpod does not remove any
+Kubernetes objects, such as your TLS certificate or connection secrets.
+
+```shell
+kubectl create namespace gitpod
+```
+
+# Todo
```
Comment (MrSimonEmms) [inline install/installer/docs/overview.md:584]: This subheading and the stuff below may as well be removed now - not the end of the world if it's not though

**Review by MrSimonEmms — APPROVED** [review]
/hold

One comment - feel free to ignore

File: install/installer/docs/overview.md (hunk 2 above)
Comment (lucasvaltl) [reply install/installer/docs/overview.md:584]: Removing it, thanks!

**Comment by lucasvaltl** [comment]
Resolved all suggestions / conversations. 

/unhold

**Pull request merged**